gnucash/register/ledger-core/split-register-copy-ops-internals.c

```diff
@@ -0,0 +1,455 @@
+/********************************************************************\
+ * split-register-copy-ops-internals.c -- internal details of       *
+ *                 copy/paste semantics for transactions and splits *
+ * Port to C of engine-interface                                    *
+ * originally written by Dave Peticolas <dave@krondo.com>           *
+ * © 2019 Geert Janssens
+ *                                                                  *
+ * This program is free software; you can redistribute it and/or    *
+ * modify it under the terms of the GNU General Public License as   *
+ * published by the Free Software Foundation; either version 2 of   *
+ * the License, or (at your option) any later version.              *
+ *                                                                  *
+ * This program is distributed in the hope that it will be useful,  *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the    *
+ * GNU General Public License for more details.                     *
+ *                                                                  *
+ * You should have received a copy of the GNU General Public License*
+ * along with this program; if not, write to the Free Software      *
+ * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.        *
+\********************************************************************/
+
+#include <glib.h>
+
+#include "config.h"
+#include "split-register-copy-ops-internals.h"
+
+/* constructor */
+floating_split *gnc_make_float_split (void)
+{
+    return g_new0(floating_split, 1);
+}
+
+/* type predicate */
+gboolean is_gnc_float_split (const floating_split *fs)
+{
+    if (fs)
+        return TRUE;
+    else
+        return FALSE;
+}
```

Comment:
Eww. `return fs != NULL`.

Reply:
Heh, indeed. I was in minimal transliteration mode apparently. Fixed.